src/rubicon/objc/types.py

```diff
@@ -799,6 +805,14 @@ class NSRect(Structure):
         ("size", NSSize),
     ]
 
+    def __repr__(self):
+        return f"<NSRect({self.size.width}x{self.size.height} @ {self.origin.x}, {self.origin.y})>"
```

Comment:
Here we've got a size (which doesn't need brackets), and a point (which does, but only on the "pointy" part):
```suggestion
        return f"<NSRect {self.size.width}x{self.size.height} @ ({self.origin.x},{self.origin.y})>"
```